### DISCUSSION TITLE
关于对逐火英桀第一位——“凯文凯文·卡斯兰娜”称号“救世”的宣传报告

### DISCUSSION THREAD

**Discussion by azb-666** [share121/inter-knot#3036] [其他]
上传不了是一款我的问题，总之是我的问题，所以图要重放试试看了

![凯文](https://github.com/user-attachments/assets/ba60145b-f133-4f98-9aa3-40fa6ef73c79)

**Comment by 1ctrl-cv** [comment]
然而你的图没传上来

**Reply by azb-666** [reply]
我的问题，现已重现上传

**Comment by XYdtman** [comment]
<img src="https://img-static.mihoyo.com/communityweb/upload/297ba8e3b6b7509d12230043310c3cc2.png" alt="凯文-气" width="75" >想你了，牢文

**Comment by Nie7bai** [comment]
所以，鸟为什么会飞？

**Comment by MaaY88933** [comment]
蒸！<img src="https://bbs-static.miyoushe.com/static/2023/11/07/feb46dc482bc621cc68058b0f77478d6_3301492614879353942.png" alt="桂乃芬 惊" width="75" >
